topics/asciidoc.adoc

```diff
@@ -57,11 +64,61 @@ include::common/attributes/community.adoc
 ----
 +
 
-The `ifdef` statement must be at the start of a new line, even for inline references. When the *Community Documentation* version of the guide is rendered, the content inside the `ifdef` statements for `community` will appear, and the content inside the `ifdef` statements for `productx` will be excluded. Any content not included inside an `ifdef` statement will appear in rendered guides for all brands.
+The `ifdef` statement must be at the start of a new line, even for inline references.
+When the *Community Documentation* version of the guide is rendered, the content inside the `ifdef` statements for `community` will appear, and the content inside the `ifdef` statements for `productx` will be excluded.
+Any content not included inside an `ifdef` statement will appear in rendered guides for all brands.
 
 . Render the guide to verify that your attributes are appearing correctly.
+Navigate to the directory containing the `master.adoc`, and use:
++
+[source,bash]
+--
+asciidoctor compile
+--
+
+[ccg-attributes-to-links]
+=== Single-sourcing URLs using attributes
+
+In addition to storing product names and versions, attributes can be used to store book-local and cross-book links for your guides.
+Doing this allows you to:
+
+* construct consistently structured and predictable link URLs (especially when combined with explicit ID anchors),
+* single-source your link URLs, making updates easier,
+* eliminate the risk of introducing errors when updating your links.
+
+To store a relative link as an attribute, you must define it in the:
+
+[source,asciidoc,option="nowrap"]
+--
+:link-getting-started: /docs/getting-strated.html
+--
+//Next step: Specify the variable in a link macro.
+
+Attributes can also store external link URLs, allowing you to update them without breaking any links in your guide.
++
+[source,asciidoc,options="nowrap"]
+--
+:link-community-docs: https://docs.example.com/guides/community.html
+--
+
+Use the variable in a link macro.
+The variable will automatically resolve to the URL defined in the attribute when you build your documents.
+
+[source,asciidoc,options="nowrap",subs="attributes+"]
+--
+link:{link-community-docs}#introduction[Community Documentation Introduction^]
+--
+TIP: Use the caret `^` character to enforce the `target=_blank` attribute for the link anchor.
```

Comment:
Actually this isn't true, it's a combination of including the :linkattrs: value and using the caret (http://asciidoctor.org/docs/asciidoc-writers-guide/#links-and-images)

Reply:
@ncbaratta Thank you, You're right, without `linkattrs`, it wouldn'y work. Let me add that information.